keyboards/misterknife/knife66/readme.md

```diff
@@ -0,0 +1,11 @@
+## Introduction
```

Comment:
Should follow the template, <https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md>